lib/paper_trail/record_trail.rb

```diff
@@ -237,7 +237,10 @@ def record_create
       @in_after_callback = true
       return unless enabled?
       versions_assoc = @record.send(@record.class.versions_association_name)
-      versions_assoc.create! data_for_create
+      version = versions_assoc.new data_for_create
+      version.item_type = @record.class.name
```

Comment:
Can `item_type` be moved into `data_for_create`?

Reply:
Unfortunately when initializing the new Version, ActiveRecord uses the polymorphic association to override whatever you specify for `item_type`, so we need to `new` it and then reset the `item_type` on our own.